.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "third_party/xsimd"]
 	path = third_party/xsimd
 	url = https://github.com/xtensor-stack/xsimd.git
+[submodule "third_party/simdjson"]
```

Comment:
Can we have simdjson via cmake resolve_dependency() ? 
This allows for a lot more flexibility and are also planning to move gtest/xsimd to use resolve_depedency ?

Reply:
cc : @assignUser

Reply:
Yep, this should be no problem to convert  as the submodule builds without problems so it's just a question of adding the right url and sha to the template. The module also has a readme that explains how to do it now!